pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = [ "setuptools", "setuptools_scm" ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ts_m2"
+description = """
+This package is the commandable SAL component (CSC) and provides a simple
+emulator for the M2 control system."""
+license = { text = "GPL" }
+classifiers = [ "Programming Language :: Python :: 3" ]
+urls = {documentation = "https://jira.lsstcorp.org/secure/Dashboard.jspa", repository = "https://github.com/lsst-ts/ts_m2" }
+dynamic = [ "version" ]
+
+[tool.setuptools.dynamic]
+version = { attr = "setuptools_scm.get_version" }
+
+[tool.setuptools.packages.find]
+where = [ "python" ]
+
```

Comment:
Please add the entry_points here as well.

Reply:
Added in the commit: [dc78758](https://github.com/lsst-ts/ts_m2/pull/25/commits/dc78758f743d2867d90bbb8500248fe6606a8870). Thanks!